geoips/commandline/geoips_command.py

```diff
@@ -294,9 +294,18 @@ def _get_registry_by_interface_and_package(self, interface, package_name):
               ("module_based", "yaml_based", "text_based")
         """
         if package_name == "all":
-            interface_registry = interface.plugin_registry.registered_plugins[
-                interface.interface_type
-            ][interface.name]
+            # If there are no plugins of current interface,
+            # just return None, do not fail catastrophically.
+            # This will fail on "sector_adjusters" interface
+            # during "geoips list plugins" if only geoips
+            # repo is installed (since there are no "sector_adjuster"
+            # plugins in the geoips repo)
+            if interface.name in interface.plugin_registry.registered_plugins[interface.interface_type]:
+                interface_registry = interface.plugin_registry.registered_plugins[
+                    interface.interface_type
+                ][interface.name]
+            else:
+                return None
```

Comment:
This caused "geoips list plugins" to fail - so probably the one we *do* want to fix.

Reply:
Note these need black applied .... Sorry, can't push updates right now.

Reply:
It did? This does not fail on my end. What packages do you have installed?